bindings/python/test/estimator/test_orbit_determination_solver.py

```diff
@@ -106,7 +126,7 @@ def termination_criteria() -> str:
 
 
 @pytest.fixture
-def estimated_state() -> State:
+def estimate() -> State:
```

Comment:
```suggestion
def estimated_state() -> State:
```
If we're sticking to estimated_state for this in python